config/manager/manager.yaml

```diff
@@ -70,13 +61,19 @@ spec:
         - /manager
         args:
         - --leader-elect
+        - --remote-cluster-config-file
```

Comment:
Will this get overwritten the next time we push?

Reply:
I removed it. Now it's generated in the workflow.